### PR TITLE
Proposal: rename the "master" brand

### PR DESCRIPTION
**Pull request by notlee**
👉 [view proposal](https://github.com/Financial-Times/origami/blob/proposal-rename-master-brand/proposals/accepted/0000-rename-master-brand.md)

### CODE REVIEW

File: proposals/accepted/0000-rename-master-brand.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- [`create-origami-component`](https://github.com/Financial-Times/create-origami-component/): component boilerplate uses the master brand
+- [origami-website](https://github.com/Financial-Times/origami-website/): documentation references the master brand
+
+To remove internal master brand references:
+
+- Update each component to upgrade to the latest release of `o-brand` and replace references in Sass and `origami.json`.
+- Update [`origami-build-tools`](https://github.com/Financial-Times/origami-build-tools) SassDoc and tests
+- Update [`origami-codedocs`](https://github.com/Financial-Times/origami-codedocs) `@brand` annotation tests
+- A new version of the [`origami-repo-data`](https://github.com/Financial-Times/origami-repo-data) api to no longer accept the master brand in api requests and to normalise the brand returned for existing component releases. The latter would allow the brand alias to be removed from [`origami-registry-ui`](https://github.com/Financial-Times/origami-registry-ui).
+- A new version of the [`origami-build-service`](https://github.com/Financial-Times/origami-build-service) api to no longer accept the master brand.
```
Comment (notlee) [inline proposals/accepted/0000-rename-master-brand.md:39]: this isn't necessary, the alias could stay

File: proposals/accepted/0000-rename-master-brand.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+>
+>Is this something worth considering when the opportunity comes up to review them?
+
+https://financialtimes.slack.com/archives/C02FU5ARJ/p1592311869388000
+
+## work required
+
+I believe the master brand originally came from the design team. To avoid confusion we would need to communicate with other teams to ensure we all refer to the master brand by its new name.
+
+As the default brand, renaming the master brand effects quite a few projects. Tools and services would need to introduce an alias to support current component and future releases; `o-brand` would need a major release; and all branded components would require Sass and `origami.json` updates, but that could happen over time.
```
Comment (notlee) [inline proposals/accepted/0000-rename-master-brand.md:19]:
```suggestion
As the default brand, renaming the master brand effects quite a few projects. Tools and services would need to introduce an alias to support current and future component releases; `o-brand` would need a major release; and all branded components would require Sass and `origami.json` updates, but that could happen over time.
```

File: proposals/accepted/0000-rename-master-brand.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# put your proposal title here
```
Comment (chee) [inline proposals/accepted/0000-rename-master-brand.md:1]:
```suggestion
# Rename the master brand
```

**Comment by chee** [comment]
my vote is for the new name to be `Pink`

**Comment by notlee** [comment]
☝️Plus one to "pink", [we and others](https://financialtimes.slack.com/archives/CSW6B2VAN/p1616500932023500) often find ourselves explaining the master brand by calling it "the pink one". Also pink puts it on a level with other brands in terms of importance, the other brands aren't an after thought

File: proposals/accepted/0000-rename-master-brand.md (hunk 1 above)
Comment (notlee) [reply proposals/accepted/0000-rename-master-brand.md]: Spec v2 / npm only components changes this, we're already releasing a new version of the build service soon, if we can agree on a name with the broader design team quickly now would be a good time. An alias in repo data to support old and new components would be the only alias we'd need.

**Comment by notlee** [comment]
@Financial-Times/origami-core I've updated this proposal to account for the NPM milestone and to include the new name suggestions.

**Comment by notlee** [comment]
Proposal moved to issue https://github.com/Financial-Times/origami/issues/243

**Pull request closed without merging**